validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 11 | Verify default storage class creation     |      | `longhorn` storage class exists and is available |
+
+### Longhorn Installation with Custom Configuration
+
+Test Longhorn installation with custom settings including replica count, data path, and node selection through Rancher interface.
+
+| Step Number | Action | Data | Expected Result                             |
+|-------------|--------|--------|----------------------------------------------|
+| 1           | Access Longhorn installation chart in Rancher Apps | | Longhorn installation page displays         |
+| 2           | Click "Longhorn Default Settings" to expand configuration | | Configuration options become visible         |
+| 3           | Set Default replica count to `2` | | Setting accepts value of 2                   |
+| 4           | Set Default data path to `/var/lib/longhorn-custom` |  | Custom path is configured                    |
+| 5           | Set Storage over-provisioning percentage to `150` |  | Over-provisioning set to 150%                |
+| 6           | Enable "Create default disk only on labeled node" | | Option is enabled                            |
+| 7           | Label worker nodes with `node.longhorn.io/create-default-disk=true` | | Nodes are properly labeled                  |
+| 8           | Complete installation with custom configuration | | Installation succeeds with custom settings   |
+| 9           | Verify custom settings in Longhorn UI | | Settings reflect custom configuration        |
+| 10          | Create test volume and verify replica count | | Volume created with 2 replicas               |
+| 11          | Check custom data path utilization on nodes | | `/var/lib/longhorn-custom` directory is used |
+
+### Longhorn UI Access Through Rancher
+
+Verify seamless access to Longhorn UI through Rancher interface with proper authentication and session management.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Navigate to cluster view in Rancher dashboard | | Cluster overview displays |
+| 2 | Verify "Longhorn" appears in left navigation menu | | Longhorn menu item is visible |
+| 3 | Click "Longhorn" menu item | | Longhorn resources display in Rancher |
+| 4 | Click the "Longhorn" button in the Overview section | | Longhorn UI opens in new tab/window |
+| 5 | Navigate to "Volume" section in Longhorn UI | | Volume management page loads |
+| 6 | Navigate to "Node" section in Longhorn UI | | Node management page loads |
+| 7 | Navigate to "Setting" section in Longhorn UI | | Settings page loads |
+| 8 | Return to Rancher tab and verify session maintained | | Rancher session still active |
+| 9 | Create test volume from Longhorn UI | | Volume creation succeeds |
+| 10 | Return to Rancher and verify volume appears | | Volume visible in Rancher Longhorn section |
+
+### RBAC Integration Testing
```

Comment:
I would vote for each of these to be their own tests vs. all in one test in qase.